### ISSUE TITLE
Is it possible that Huggingface Transformers are ported to C++ using Flashlight?

### ISSUE THREAD

**Issue by QMrpy** [flashlight/flashlight#720]
### Question
Is it possible to implement the Transformers library using Flashlight? Is such a feature planned to be released in the future?

**Comment by tlikhomanenko** [comment]
Yep, you can implement Transformer library with Flashliligh and we already have several implementations for it. What do you mean exactly by "Is such a feature planned to be released in the future" (my understanding of huggingface is that it contains a lot of implementations of papers - which is not really library, library is more about benchmarks)?
